repository/repository.go

```diff
@@ -0,0 +1,9 @@
+package repository
+
+import "database/sql"
+
+type repository struct {
+	DB *sql.DB
+}
+
+var Repository repository
```

Comment:
Better to pass this to every handler, no?

Reply:
how would you implement those?
so instead of `chi.Route()` use `chi.Handle()` instead?

Reply:
Okay, let me think of a moment.

Reply:
I guess we can do this on the next PR.